### PR TITLE
css books on shelf

### PR DESCRIPTION
**Pull request by gabrielbursztein**
Css in Books on my shelf.

### CODE REVIEW

File: app/controllers/requests_controller.rb [hunk 1]
```diff
@@ -1,7 +1,11 @@
 class RequestsController < ApplicationController
 
   def index
-    @requests = current_user.requests
+    if user_signed_in?
```
Comment (GuilleLeopold) [inline app/controllers/requests_controller.rb]: You can write this if sentence in one line: 
`user_signed_in? ? @requests = current_user.requests : redirect_to new_user_session_path`

File: app/views/books/index.html.erb [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+  <table>
+    <% @books.each do |book| %>
+      <tr>
+        <td><%= book.title %></td>
+        <td><%= book.author %></td>
+        <td><%= book.year %></td>
+        <% if user_signed_in? %>
+          <% if !current_user.requested?(book) %>
+            <td id='<%= book.id %>'>
+              <%= link_to 'Request', requests_path(:book_id => book.id), method: :post, remote: true, class: "bold underlined" %>
```
Comment (GuilleLeopold) [inline app/views/books/index.html.erb]: `:book_id => book.id `syntax is deprecated, use the new syntax: `book_id: book.id`

**Review by GuilleLeopold — COMMENTED** [review]
Same as the PR before. I don't know if @MaicolBen can check the css, if not, add Ale or Alexis as a reviewer.

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -13,3 +13,128 @@
  *= require_tree .
  *= require_self
  */
+
+.wrapper {
```
Comment (MaicolBen) [inline app/assets/stylesheets/application.css]: Don't put code here, move it to another file

File: app/assets/stylesheets/application.css [hunk 4]
```diff
@@ -13,3 +13,128 @@
...
+  font-weight: 900;
+  text-align: center;
+}
+
+.my_books {
+  margin-top: 20px;
+}
+
+.book_title {
+  font-family: "Times New Roman";
```
Comment (MaicolBen) [inline app/assets/stylesheets/application.css]: Use font variable that is already defined

File: app/assets/stylesheets/application.scss [hunk 5]
```diff
@@ -89,16 +86,58 @@ a.request {
...
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  a {
+    color: black;
+    font-weight: bold;
+    font-size: 15px;
+    text-decoration: none;
+  }
+
```
Comment (MaicolBen) [inline app/assets/stylesheets/application.scss]: Remove empty lines

**Review by MaicolBen — CHANGES_REQUESTED** [review]
Add `db/development.sqlite3` to the git ignore

**Review by MaicolBen — APPROVED** [review]

**Pull request merged**